pkg/management/component.go

```diff
@@ -1795,6 +1801,8 @@ func (c *component) UpdateAuthorizations(ctx context.Context, realmName string,
 		}
 		c.logger.Info(ctx, "msg", "Updated authorizations", "add_count", len(addAuthz), "del_count", len(delAuthz))
 
+		c.kafkaProducer.SendMessageBytes([]byte{})
```

Comment:
Use `_ = c.kafkaProducer.SendMessageBytes([]byte{})` or check the error

Reply:
here you should clearly handle the error case